src/c++/perf_analyzer/load_worker.cc

```diff
@@ -500,7 +537,7 @@ LoadWorker::SendRequest(
           callback_func, *(context->options_), context->valid_inputs_,
           context->outputs_);
     }
-    context->inflight_request_cnt_++;
+    total_ongoing_requests_++;
```

Comment:
It looks like the before was context-specific. Can you elaborate on why this change was made?

Reply:
OLD: Concurrency code used total_ongoing_requests, while RequestRate code used inflight_request_cnt to track how many were outstanding so it could ensure that all had finished before entering certain parts of code.

NEW: I standardized the tracking so they both use the same variable. The request rate worker only ever has a single context.  You can see that `GetCtxId()` is hardcoded to 0 for that class.